util/site/fetch_block_stats.py

```diff
@@ -27,7 +27,7 @@
     "csrng": "hw/ip/csrng/dv/2024.01.22_10.48.06/",
     "edn": "hw/ip/edn/dv/2024.01.22_10.48.55/",
     "entropy-source": "hw/ip/entropy_src/dv/2024.01.22_10.49.35/",
-    "flash": "hw/ip/flash_ctrl/dv/2024.01.22_10.51.11/",
+    "flash": "hw/top_earlgrey/ip_autogen/flash_ctrl/dv/2024.01.22_10.51.11/",
```

Comment:
This is the same issue as the nightly results -- The date points to results that won't exist.